examples/google_hangouts_bot.js

```diff
@@ -0,0 +1,95 @@
+var Botkit = require('../lib/Botkit.js');
+
+var controller = Botkit.googlehangoutsbot({
+    endpoint: 'Axjn86rTGRQwisaYFyT0XZyiOCh7rZUPGx1A',
+    token: "YOUR_TOKEN",
+    debug: true,
+});
+
+var bot = controller.spawn({});
+
+controller.setupWebserver(3000, function (err, webserver) {
+    controller.createWebhookEndpoints(webserver, bot, function () {
+        console.log(`🚀 Congratulation, the web server is online !`);
```

Comment:
In English, there is no space in front of the punctuation marks ; ! ? :

Reply:
Done